README.md

```diff
@@ -1,6 +1,5 @@
 [[⬇️ **Download**]](https://github.com/containerd/stargz-snapshotter/releases)
 [[📔**Browse images**]](./docs/pre-converted-images.md)
-[[☸**Quick Start (Kubernetes)**]](#quick-start-with-kubernetes)
```

Comment:
Please don't remove this.

Reply:
If keep Kubernetes , I think we need to add Systemd

Reply:
How about adding a new link something like "**Install**" that links to INSTALL.md?